### ISSUE TITLE
Lint against Composition Locals that use `get() =` syntax

### ISSUE THREAD

**Issue by ZacSweers** [slackhq/compose-lints#218]
This should be a big error

```kotlin
internal actual val LocalThing:
  ProvidableCompositionLocal<Thing>
  get() = compositionLocalOf { Thing() }
```

vs

```kotlin
internal actual val LocalThing:
  ProvidableCompositionLocal<Thing> = compositionLocalOf { Thing() }
```
